### PR TITLE
Update exposed service styling

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3129/
Test FAILed.

**Comment by anthonydillon** [comment]
### QA
- Drop a service on the canvas
- Commit it and expose it
- See that the border of the service is 4px width
- Unexpose the service
- Check that the stroke-width has returned to normal

File: jujugui/static/gui/src/app/assets/css/base.scss [hunk 1]
```diff
@@ -653,17 +653,25 @@ g.service {
...
         fill: #4c4c4c;
     }
+
     .name {
       fill: rgb(80, 80, 80);
       font-size: 14px;
     }
+
+    .service-block {
+        transition: all .3s;
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/assets/css/base.scss]: Quick transition between states as a nice drive by.

Comment (hatched) [inline jujugui/static/gui/src/app/assets/css/base.scss]: This makes the whole branch :)

**Comment by hatched** [comment]
:+1: QA :ok: 
Looks awesome

**Comment by makyo** [comment]
:+1: awesome, thanks!

**Comment by hatched** [comment]
CI Passed when run manually http://ci.jujugui.org:8080/job/juju-gui/3133

**Comment by anthonydillon** [comment]
Thanks @hatched and @makyo :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**